improver/cube_combiner.py

```diff
@@ -244,6 +253,22 @@ def _get_expanded_coord_names(cube_list: Union[List[Cube], CubeList]) -> List[st
                     expanded_coords.append(coord)
         return expanded_coords
 
+    def _add_cell_method(self, cube: Cube) -> None:
+        """Add a cell method to record the operation undertaken.
+
+        Args:
+            cube:
+                Cube to which a cell method will be added.
+        """
+        cell_method_lookup = {"max": "maximum", "min": "minimum", "mean": "mean"}
+        if self.operation in ["max", "min", "mean"] and self.cell_method_coordinate:
+            cube.add_cell_method(
+                CellMethod(
+                    cell_method_lookup[self.operation],
+                    coords=self.cell_method_coordinate,
+                )
+            )
```

Comment:
This currently doesn't raise a warning / exception if the user tries to add a cell method in combination with an operation that doesn't support it, e.g. subtract. I think it should raise a flat out exception so the user is in no doubt about what they've done and why no cell method is appearing on their output.

Reply:
I've altered this so that an exception is raised if a `cell_method_coordinate` is passed but the operation is not max, min or mean.